.java-version

```diff
@@ -0,0 +1 @@
+graalvm64-11.0.20
```

Comment:
we need to exclude this file and add a new entry to gitignore. 
The file is produced by `jenv`